packages/iTwinUI-react/src/types/toaster.ts

```diff
@@ -0,0 +1,4 @@
+export interface Root {
```

Comment:
Why not just keep this in the `Toaster` file? I think the `types/` directory is only for declarations (`.d.ts` files)

Reply:
got rid of it